tardis/gui.py

```diff
@@ -1,5 +1,12 @@
 import numpy as np
 import matplotlib
+matplotlib.style.use('fivethirtyeight')
```

Comment:
The style packet has been added to matplotlib starting with version 1.4:

http://matplotlib.org/users/whats_new.html#style-package-added

To make the gui work also with older versions, maybe add a check which only uses this style when matplotlib >= 1.4 is available.